code/datums/datum.dm

```diff
@@ -102,7 +102,9 @@
  */
 /datum/proc/Destroy(force=FALSE, ...)
 	SHOULD_CALL_PARENT(TRUE)
+#ifndef UNIT_TESTS
```

Comment:
Please add a comment explaining why, in this case it's because unit tests allow destroy to yield while it finds the references of what is getting it, or it might be the input prompt asking if you want to do that? I forget which. On that note it might be better to use the define for *that*, since UNIT_TESTS just flips it on. I defer to @LemonInTheDark

Reply:
Different approach